### ISSUE TITLE
Android In-App Messaging: local notification is not sent, when Adaptive icon is used as application icon

### ISSUE THREAD

**Issue by RobertZagorski** [Adobe-Marketing-Cloud/mobile-services#247]
Prequisities:
Android Application is targeting Android API 26
In [mobile marketing console](https://mobilemarketing.adobe.com) **In-app message** is set to show **Local notification** on any event.

I am using [Adaptive Icon](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive.html) as application icon on API 26 device. On lower devices application icon is .png.
When SDK is trying to send Local notification through [Notification.Builder](https://developer.android.com/reference/android/app/Notification.Builder.html), the following message is printed in console:

>E/ADBMobile: ADBMobile Error : Messages - unexpected error posting notification (android.graphics.drawable.AdaptiveIconDrawable cannot be cast to android.graphics.drawable.BitmapDrawable)

Problematic code (from decompiled by Android Studio library):
_com.adobe.mobile.MessageNotificationHandler.class:197_
```
Bitmap icon = null;
if(iconDrawable != null) {
     icon = ((BitmapDrawable)iconDrawable).getBitmap();
}
```
Hope the problem is understoodable.
I am aware of `Config.setLargeIconResourceId(...)` method, however I would not like to add another  _.png_ in application resources.

**Comment by shalehaha** [comment]
@RobertZagorski,

Thanks for reporting this, I've logged a feature request in our internal system. We will fix this in a future release and will update this ticket once we have a date for it. 
But before that, I believe you have to use the Config.setLargeIconResourceId(...) API.

**Comment by shalehaha** [comment]
Closing the ticket, it had already been fixed in version 4.15.0.
